src/current/_includes/releases/v23.2/v23.2.9.md

```diff
@@ -0,0 +1,75 @@
+## v23.2.9
+
+Release Date: August 1, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-9-sql-language-changes">SQL language changes</h3>
+
+- [`EXPLAIN ANALYZE`]({% link v23.2/explain-analyze.md %}) statements are now supported when executed via Cloud Console [SQL shell]({% link cockroachcloud/sql-shell.md %}). [#125562][#125562]
+- Added the [`sql.auth.grant_option_inheritance.enabled` cluster setting]({% link v23.2/cluster-settings.md %}#setting-sql-auth-grant-option-inheritance-enabled). The default value is `true`, which maintains consistency with CockroachDB's previous behavior: users granted a privilege with [`WITH GRANT OPTION`]({% link v23.2/grant.md %}) can in turn grant that privilege to others. When `sql.auth.grant_option_inheritance.enabled` is set to `false`, the `GRANT OPTION` is not inherited through role membership, thereby preventing descendant roles from granting the privilege to others. However, the privilege itself continues to be inherited through role membership. [#126298][#126298]
+
+<h3 id="v23-2-9-operational-changes">Operational changes</h3>
+
+- `crdb_internal.cluster_execution_insights.txt` and `crdb_internal.cluster_txn_execution_insights.txt` have been removed from the [debug zip]({% link v23.2/cockroach-debug-zip.md %}). These files contained cluster-wide insights for statements and transactions. Users can still rely on the [per-node execution]({% link v23.2/cockroach-debug-zip.md %}#files) insights in `crdb_internal.node_execution_insights.txt` and `crdb_internal.node_txn_execution_insights.txt`. [#125804][#125804]
+- Some debugging-only information about physical plans is no longer collected in the `system.job_info` table for changefeeds, because it has the potential to grow very large. [#126097][#126097]
```

Comment:
```suggestion
- Some debugging-only information about physical plans is no longer collected in the `system.job_info` table for [changefeeds]({% link v23.2/change-data-capture-overview.md %}), because it has the potential to grow very large. [#126097][#126097]
```

Reply:
added link